sdks/python/apache_beam/ml/inference/base_test.py

```diff
@@ -319,6 +326,24 @@ def mult_two(example: str) -> int:
     with self.assertRaises(ValueError):
       base.KeyedModelHandler(mhs)
 
+  def test_keyed_model_handler_get_num_bytes(self):
+    mh = base.KeyedModelHandler(FakeModelHandler(num_bytes_per_element=10))
+    batch = [('key1', 1), ('key2', 2), ('key1', 3)]
+    expected = len(pickle.dumps(('key1', 'key2', 'key1'))) + 30
+    actual = mh.get_num_bytes(batch)
+    self.assertEqual(expected, actual)
+
+  def test_keyed_model_handler_multiple_models_get_num_bytes(self):
+    mhs = [
+        base.KeyMhMapping(['key1'], FakeModelHandler(num_bytes_per_element=10)),
```

Comment:
do you still plan to change the name of KeyMhMapping?

Reply:
Yes, I'm waiting until I don't have in flight PRs around this to change it to avoid conflicts (right now https://github.com/apache/beam/pull/28026 uses KeyMhMapping)